Gemfile.lock

```diff
@@ -101,7 +136,8 @@ DEPENDENCIES
   rubocop-rspec (~> 2.12.1)
   rubocop-sorbet (~> 0.6.0)
   sorbet (~> 0.5.9204)
+  tapioca (~> 0.9.4)
   yard-sorbet!
 
 BUNDLED WITH
-   2.3.20
+   2.3.13
```

Comment:
Downgraded the bundler version to keep platform specific versions of `sorbet-static` until a version with https://github.com/rubygems/rubygems/pull/5832 is released.

Reply:
I've kept this, but just fyi, when I run `bundle update` locally with bundler 2.3.13 i get this error: https://github.com/rubygems/rubygems/issues/5088

Reply:
Interesting. It's working for me on Ruby `3.1.2`. Hopefully we can bump Bundler soon.